imap_processing/lo/l1a/lo_l1a_write_cdfs.py

```diff
@@ -121,3 +126,158 @@ def create_lo_scide_dataset(sci_de: list):
     )
 
     return sci_de_dataset
+
+
+def create_lo_scicnt_dataset(sci_cnt: list):
+    """
+    Create Lo L1A Science Counts Dataset from the ScienceCounts dataclasses.
+
+    Parameters
+    ----------
+    sci_cnt: list
+        List of ScienceCounts data class objects.
+
+    Returns
+    -------
+    xarray.Dataset
+        Lo L1A Science Counts Dataset.
+    """
+    # TODO: How should the times be computed? Using START_A + SHCOARSE for now
+    sci_cnt_times = np.concatenate(
+        [sci_cnt_data.START_A + sci_cnt_data.SHCOARSE for sci_cnt_data in sci_cnt]
```

Comment:
START_A is 6x7 matrix, this doesn't make sense. I'll use SHCOARSE only for now. Still need to get more info from Lo on how to store the epoch.